paddle/fluid/framework/ir/mkldnn/params_quantization_mkldnn_pass.cc

```diff
@@ -170,8 +212,10 @@ void ParamsQuantizationMkldnnPass::ApplyImpl(ir::Graph* graph) const {
                           platform::errors::InvalidArgument(
                               "Pointer to graph argument should not be NULL."));
   FusePassBase::Init(name_scope_, graph);
-  QuantizeConv(graph, true /*with_residual_data*/);
-  QuantizeConv(graph, false /*with_residual_data*/);
+  QuantizeConv(graph, "fused_conv2d", true /*with_residual_data*/);
+  QuantizeConv(graph, "fused_conv2d", false /*with_residual_data*/);
+  QuantizeConv(graph, "conv2d", true /*with_residual_data*/);
+  QuantizeConv(graph, "conv2d", false /*with_residual_data*/);
```

Comment:
This pass works on convolution ops that had scales calculated during int8_scales_calculation_pass. You have added the code linked below that changes conv2d to fused_conv2d there here https://github.com/PaddlePaddle/Paddle/pull/48848/files?show-viewed-files=true&file-filters%5B%5D=#diff-202dc497cf83197c83edced692ade07f943a14605f0b2cefc9f10a36a244212bR126-R127.
So we should be able to safely assume it can work only on fused_conv2d.

For the quantization of weights to happen the fused_conv2d needs Scale_weights attribute to be present. If conv op has bias, then bias is also scaled according to Bias_scales attribute.

Reply:
For the quantization of weights to happen the fused_conv2d needs `Scale_weights `attribute to be present. If conv op has bias, then bias is also scaled according to `Bias_scales `attribute.

Reply:
Thank you, I think you are right. The `conv2d` in `params_quantization_mkldnn_pass` has been removed.